bears/setup.py.in

```diff
@@ -23,6 +23,9 @@ if __name__ == "__main__":
               license='$LICENSE',
               packages=find_packages(exclude=["build.*"]),
               long_description="""$LONG_DESCRIPTION""",
+              entry_points={"$BEAR_NAME": [
```

Comment:
nah this has to be `coalabears` for it to work, I don't know if it's documented anywhere